README.md

```diff
@@ -511,12 +520,22 @@ Returns the method associated with current auth flow; it might be sms, otp or pu
 ### Enrollment
 
 #### enrollment.getAvailableMethods()
+Returns an array of strings that represent the authentication methods that can
+be used for this enrollment in the current transaction (the available methods
+might change from transaction to transaction). They depends on the methods supported
+by the enrollment and the methods currently active.
```

Comment:
"…and the methods currently allowed by the tenant." I think could be clearer, for people not knowing if an enrollment could have inactive methods or not.